### ISSUE TITLE
[TSDoc Home page]: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds.[Sev2][WCAG1.4.3][Auto]

### ISSUE THREAD

**Issue by SrujanaAnaganti** [microsoft/tsdoc.org-website#9]
**Test Environment:**
OS Build: Windows 10
Browser: Edge Dev
URL: https://tsdoc.org/

**Repro Steps:**
1) Open the above URL and login with valid credentials.
2) TSDoc page will get displayed.
3) Now run Accessibility Insights for web tool and observe the issue.

**Actual:**
Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds for 'Intro' , 'Tags', 'GitHub' and 'Help' links. These elements have insufficient contrast ratio of 3.94.

**Expected:**
Contrast ratio for the text must be greater than or equal to 4.5:1

**User Impact:**
When a minimum contrast ratio is not provided between text and its background, it is difficult for low-vision users as well as anyone in extreme lighting environments to read the content.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum
<img width="960" alt="Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds_homepage" src="https://user-images.githubusercontent.com/85154417/121009635-0b24cf80-c7b2-11eb-8be5-6dad1416a849.png">
